components/leaflet-maps/ClustersMap.vue

```diff
@@ -195,7 +195,53 @@
        })
 
        return optionalsWithValue
-     }
+     },
+     getFormatDate: function (date,precision){
+
+      let [day, month, year] = date.split('/');
+      day = Number(day);
+      month = Number(month)
+      year = Number(year);
+      let currentDate = new Date(year, month - 1, day);
+      
+      
+      //the precision of the dates can either be or 9 11(full day/month/year),10(month/year),9(year)
```

Comment:
Instead of encoding these in a comment, it would be better to use some consts e.g.

```
const PRECISION_DAY_MONTH_YEAR = 11;
const PRECISION_MONTH_YEAR = 10;
const PRECISION_YEAR = 9
```

Because these are verbosely named it should make your code easier to read as well. Thanks Ruby

Reply:
Changed. Looks good. Thank you